pbjam/priors.py

```diff
@@ -149,8 +158,13 @@ def _prior_size_check(self, pdata, numax, KDEsize):
             idx = np.abs(pdata.numax.values - numax[0]) < nsigma * numax[1]
 
             if not flag_warn:
+                # If this is a use warning, must give user instructions.
+                # Otherwise, make this a logger.warning
+                # Maybe user warning if len(pdata[idx]) == 0?
                 warnings.warn(f'Only {len(pdata[idx])} star(s) near provided numax. ' +
```

Comment:
I think this can be a logger warning.

Reply:
There is a unit test which looks for this warning for the case where no stars are found near numax which will fail if we change this to `logger.warning`. Should we remove that check in the unit test?